guides/release/contributing/index.md

```diff
@@ -23,6 +23,18 @@ Have a look at the [list of main repositories](repositories) to learn about the
 
 If you need some inspiration, you can check out the [Help Wanted](https://help-wanted.emberjs.com/) dashboard to browse for issues. If you are a beginner, look out for issues with the "Help wanted" and "Good first issue" labels.
 
+# Changing the API documentation
+
+To change the [API reference guidance](https://api.emberjs.com/ember/release), find the relevant section for the *package*, *method* or *class* you wish to change.
+At the top of the section, you will find a link to the 'defined in' or 'inherited from' file.
+Following this link will open the file where the guidance and the code for the function is defined.
+
+Note, the documentation is inline with the code to help ensure the code and documentation match.
+For example, near the top of the [`store.createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord) API guidance section, you can find the 'defined in' link to the specific `.ts` file where the guidance is inline: [`ds-model-store.ts`](https://github.com/emberjs/data/blob/v3.26.0/packages/store/addon/-private/system/ds-model-store.ts).
+You can then edit the guidance within the `.ts` file.
+Please follow the contributing guidance for the specific repo for example Ember.js [contributing.md](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md).
+The contributing guidance provides best practice for editing, commit messages, style and testing. 
```

Comment:
An optional suggestion. I thought the second sentence is okay, but the first sentence needed an update with regards to grammar (a run-on sentence, possibly).

```suggestion

Please follow the contributing guide in the specific repo. (Here is an example of the [contributing guide for Ember.js](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md).)
The contributing guide may further describe how you can edit files, commit messages, and run linting and testing.
```